docs/api/features/networks.md

```diff
@@ -8,6 +8,11 @@ Address Translation](http://en.wikipedia.org/wiki/Network_address_translation).
 Each default fabric network is private to a user - one user's containers cannot
 connect to another's fabric IP addresses, and vice-versa.
 
+To isolate the traffic between different applications or groups of applications,
+users may use the CloudAPI to create additional networks on their fabrics.
+Any of these user-defined networks can be designated as the 'default' network 
+to be used for provisioning Docker containers.
+
```

Comment:
We should probably add something like this to the previous paragraph like:

Note, changing the 'default' network through CloudAPI does not change any existing docker containers, it will only be used for newly created containers.